_includes/how_to_apply.md

```diff
@@ -1,3 +1,21 @@
+## How To Apply
+
+{% if page.state == 'upcoming' %}
+{% include upcoming_statement.html %}
+{% else %}
+
+{% if page.state == 'closed' %}
+We're sorry, this job has closed.
```

Comment:
Just made up a sentence here to replace the application information.

![screen shot 2018-06-05 at 5 12 01 pm](https://user-images.githubusercontent.com/86842/41003045-9eb2e2f4-68e3-11e8-9105-c4c5bdbba4f2.png)